.github/workflows/codespell.yaml

```diff
@@ -17,6 +17,7 @@ jobs:
       - name: codespell
         uses: codespell-project/actions-codespell@master
         with:
+          exclude_file: go.sum
```

Comment:
Why do we need to exclude it?

Reply:
It was failing here https://github.com/red-hat-storage/odf-operator/actions/runs/2050857294